simapp/simd/cmd/root.go

```diff
@@ -116,6 +116,10 @@ func NewRootCmd() *cobra.Command {
 	autoCliOpts := tempApp.AutoCliOpts()
 	autoCliOpts.Keyring, _ = keyring.NewAutoCLIKeyring(initClientCtx.Keyring)
 	autoCliOpts.ClientCtx = initClientCtx
+	autoCliOpts.TxConfigOpts = authtx.ConfigOptions{
```

Comment:
In the latest client/v2 we are able to read from the root context, so we can actually remove this field.

Reply:
Additionally, we already set those in the app.go when manually wiring, so this seems triple work.